### PR TITLE
Set default pricing tier to P1V2

### PR DESCRIPTION
**Pull request by Flanker32**
Set webapp default pricing tier to P1V2

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/container/webapponlinux/ui/SettingPanel.java [hunk 1]
```diff
@@ -52,6 +53,7 @@
 import com.microsoft.tooling.msservices.serviceexplorer.azure.container.WebAppOnLinuxDeployPresenter;
 import com.microsoft.tooling.msservices.serviceexplorer.azure.container.WebAppOnLinuxDeployView;
 
+import io.netty.util.internal.StringUtil;
```
Comment (Eskibear) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/container/webapponlinux/ui/SettingPanel.java]: Use `StringUtil.isBlank/isEmpty` see: http://commons.apache.org/lang/ , which is more often used. Avoid to use `*.internal.*` if possible.

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/container/webapponlinux/ui/SettingPanel.java [hunk 2]
```diff
@@ -670,11 +674,9 @@ public void renderPricingTierList(List<PricingTier> pricingTierList) {
                 cbPricing.addItem(item);
                 if (Comparing.equal(item.toString(), defaultPricingTier)) {
                     cbPricing.setSelectedItem(item);
-                    defaultPricingTier = null;
```
Comment (Eskibear) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/container/webapponlinux/ui/SettingPanel.java]: add break here.

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/webapp/webappconfig/ui/WebAppSettingPanel.java [hunk 3]
```diff
@@ -808,9 +809,11 @@ public void fillLocation(@NotNull List<Location> locations) {
     @Override
     public void fillPricingTier(@NotNull List<PricingTier> prices) {
         cbPricing.removeAllItems();
+        final String defaultPricingTier = StringUtil.isNullOrEmpty(webAppConfiguration.getPricing())
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/webapp/webappconfig/ui/WebAppSettingPanel.java]: pricingTier

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/container/webapponlinux/ui/SettingPanel.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/container/webapponlinux/ui/SettingPanel.java]: replace it with `org.jsoup.helper.StringUtil`

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/container/webapponlinux/ui/SettingPanel.java (hunk 1 above)
Comment (Flanker32) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/container/webapponlinux/ui/SettingPanel.java]: Replace it with `org.jsoup.helper.StringUtil`

**Pull request merged**